src/vpp_plugins/vpp-chaperone/VppConfigManager.cpp

```diff
@@ -790,7 +790,12 @@ void VppConfigManager::doCpeConfig (VppClient &vppClient)
             {
               LOG (INFO) << "Adding policers to CPE interface " << kv.first;
               doCpePolicerConfig (vppClient, kv.first.asString (),
-                                  kv.second["policers"]);
+                                  kv.second["policers"], true);
+            }
+          else if (!kv.second.count ("policers"))
```

Comment:
You don't need to check the same condition twice. Leave it as `else`.

Reply:
Addressed Review Comment.